reinforcement_learning/Lesson-1-Intro-to-RL-draft.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>Some minor grammatical typos throughout the notebook but that's an easy fix. This is a solid first lesson! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>